client/my-sites/domains/domain-management/list/domains-table-header.jsx

```diff
@@ -130,6 +130,7 @@ class DomainsTableHeader extends PureComponent {
 								'is-sortable': column?.isSortable,
 							} ) }
 							data-column={ column.name }
+							tabindex={ column.label ? 0 : -1 }
```

Comment:
I wonder if checking for `column?.isSortable` would be all we need here? Not sure if a header needs to be focusable if it can't be interacted with?

Reply:
Handled in [725ff59](https://github.com/Automattic/wp-calypso/pull/69489/commits/725ff593c9118b57e071dbfd822a8e15cb47afc1)